sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -1003,6 +1001,23 @@ def _predict_proba(self, X):
         normalize(distributions, copy=False, norm="l1")
         return distributions
 
+    @classmethod
+    def get_test_params(cls):
+        """Return testing parameter settings for the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        params = {
+            "random_state": 0,
```

Comment:
should the `random_state` not be controlled in the tests? E.g., if the tests want to check stochastic behaviour, this would override that. Many (refactored) tests set the `random_state`. More generally, should all tests do that?
Bit off-topic comment, perhaps sth for an issue or discussion.

Reply:
relevant fact for this PR is, I think none of the other estimators sets `random_state` in `get_test_params`, or only a small minority (consistency issue)